src/System.Windows.Forms/src/System/Windows/Forms/Control.ActiveXImpl.cs

```diff
@@ -583,7 +581,7 @@ internal unsafe HRESULT GetAdvise(DVASPECT* pAspects, ADVF* pAdvf, IAdviseSink[]
             ///  Helper function to retrieve an ambient property.  Returns false if the
             ///  property wasn't found.
             /// </summary>
-            private bool GetAmbientProperty(DispatchID dispid, ref object obj)
+            private bool GetAmbientProperty(DispatchID dispid, ref object? obj)
```

Comment:
It seems that the intent of this is to really be an out so...

```suggestion
            private bool GetAmbientProperty(DispatchID dispid, out object? obj)
```